.readthedocs.yaml

```diff
@@ -12,7 +12,7 @@ build:
     python: "3.8"
   jobs:
     post_install:
-    - pip install --upgrade pip packaging wheel setuptools==69.5.1
+    - pip install --upgrade pip wheel setuptools==69.5.1
```

Comment:
I think this part should not be required anymore.

Reply:
Okay, I wasn't sure since you seem to be pinning `setuptools` to v69.5.1 ...